### PR TITLE
[Merged by Bors] - chore: classify `simp can do this` porting notes

### PR DESCRIPTION
**Pull request by pitmonticone**
Classify by adding issue number (#10618) to porting notes claiming anything semantically equivalent to `simp can prove this` or `simp can simplify this`.

### CODE REVIEW

**Review by eric-wieser — APPROVED** [review]
I've skimmed this on the assumption it's in good faith. This looks good to me, and means that when grepping for untriaged porting notes we can exclude these.

File: Mathlib/CategoryTheory/Limits/IsLimit.lean [hunk 1]
```diff
@@ -568,7 +568,7 @@ structure IsColimit (t : Cocone F) where
 
 attribute [reassoc (attr := simp)] IsColimit.fac
 
--- Porting note: simp can prove this. Linter claims it still is tagged with simp
+-- Porting note (#10618): simp can prove this. Linter claims it still is tagged with simp
```
Comment (kbuzzard) [inline Mathlib/CategoryTheory/Limits/IsLimit.lean:571]: What is going on with this one? Is this a bug in the simpNF linter?

File: Mathlib/CategoryTheory/Limits/Shapes/Biproducts.lean [hunk 2]
```diff
@@ -258,7 +258,7 @@ structure IsBilimit {F : J → C} (B : Bicone F) where
 
 attribute [inherit_doc IsBilimit] IsBilimit.isLimit IsBilimit.isColimit
 
--- Porting note: simp can prove this, linter doesn't notice it is removed
+-- Porting note (#10618): simp can prove this, linter doesn't notice it is removed
```
Comment (kbuzzard) [inline Mathlib/CategoryTheory/Limits/Shapes/Biproducts.lean:261]: And this one!

**Comment by pitmonticone** [comment]
Thank you @eric-wieser. 

Should I include this one? 

https://github.com/leanprover-community/mathlib4/blob/83adb4cfb2c66b50fbf3eca3e520906e17fae921/Mathlib/Topology/Instances/AddCircle.lean#L173-L179

File: Mathlib/Data/Vector.lean [hunk 3]
```diff
@@ -246,7 +246,7 @@ theorem toList_mk (v : List α) (P : List.length v = n) : toList (Subtype.mk v P
 #align vector.to_list_mk Vector.toList_mk
 
 /-- A nil vector maps to a nil list. -/
-@[simp, nolint simpNF] -- Porting note: simp can prove this in the future
+@[simp, nolint simpNF] -- Porting note (#10618): simp can prove this in the future
```
Comment (kbuzzard) [inline Mathlib/Data/Vector.lean:249]: Is this the correct solution for this problem?

File: Mathlib/RingTheory/LaurentSeries.lean [hunk 4]
```diff
@@ -211,13 +211,13 @@ theorem coeff_coe (i : ℤ) :
       not_false_iff]
 #align power_series.coeff_coe PowerSeries.coeff_coe
 
--- Porting note: simp can prove this, and removed norm_cast attribute
+-- Porting note (#10618): simp can prove this, and removed norm_cast attribute
```
Comment (kbuzzard) [inline Mathlib/RingTheory/LaurentSeries.lean:214]: "and removed norm_cast attribute" seems to be an independent thing which we might also want to track?

**Review by kbuzzard — APPROVED** [review]
I've done a manual pass through this and it looks fine, but there are four instances which are not of the form "the following theorem used to be simp in lean 3 and we're removing the attribute here", which I've flagged as perhaps needing further attention.

File: Mathlib/RingTheory/LaurentSeries.lean (hunk 4 above)
Comment (pitmonticone) [reply Mathlib/RingTheory/LaurentSeries.lean:215]: @jcommelin Yes, I'll integrate it as soon as I come back home this evening.

File: Mathlib/CategoryTheory/Limits/IsLimit.lean (hunk 1 above)
Comment (pitmonticone) [reply Mathlib/CategoryTheory/Limits/IsLimit.lean:571]: This question should probably be reported in the corresponding issue #10618. 

I'm going to copy-paste it there.

Comment (pitmonticone) [reply Mathlib/CategoryTheory/Limits/IsLimit.lean:571]: Now this question can be found [here](https://github.com/leanprover-community/mathlib4/issues/10618#issuecomment-1955018632).

File: Mathlib/CategoryTheory/Limits/Shapes/Biproducts.lean (hunk 2 above)
Comment (pitmonticone) [reply Mathlib/CategoryTheory/Limits/Shapes/Biproducts.lean:261]: Now this question can be found [here](https://github.com/leanprover-community/mathlib4/issues/10618#issuecomment-1955023023).

File: Mathlib/Data/Vector.lean (hunk 3 above)
Comment (pitmonticone) [reply Mathlib/Data/Vector.lean:249]: Now this question can be found [here](https://github.com/leanprover-community/mathlib4/issues/10618#issuecomment-1955028349).

**Review by jcommelin — APPROVED** [review]
Thanks :tada:

bors merge

**Pull request closed without merging**